diff --git a/node_modules/file-saver/src/FileSaver.js b/node_modules/file-saver/src/FileSaver.js
index 5d204ae..7f33cd9 100644
--- a/node_modules/file-saver/src/FileSaver.js
+++ b/node_modules/file-saver/src/FileSaver.js
@@ -76,9 +76,8 @@ var saveAs = _global.saveAs || (
   (typeof window !== 'object' || window !== _global)
     ? function saveAs () { /* noop */ }
 
-  // Use download attribute first if possible (#193 Lumia mobile) unless this is a macOS WebView
-  : ('download' in HTMLAnchorElement.prototype && !isMacOSWebView)
-  ? function saveAs (blob, name, opts) {
+    : // Use download attribute first if possible (#193 Lumia mobile) unless this is a native app
+    (typeof HTMLAnchorElement !== "undefined" && "download" in HTMLAnchorElement.prototype)  ? function saveAs (blob, name, opts) {
     var URL = _global.URL || _global.webkitURL
     var a = document.createElement('a')
     name = name || blob.name || 'download'
